modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -1,18 +1,44 @@
 pub mod api;
 
+use bitcoin::Network;
 use fedimint_client::derivable_secret::DerivableSecret;
 use fedimint_client::module::gen::ClientModuleGen;
 use fedimint_client::module::ClientModule;
 use fedimint_client::sm::{DynState, ModuleNotifier, OperationId, State, StateTransition};
-use fedimint_client::DynGlobalClientContext;
+use fedimint_client::{Client, DynGlobalClientContext};
 use fedimint_core::core::{IntoDynInstance, ModuleInstanceId};
 use fedimint_core::db::Database;
 use fedimint_core::encoding::{Decodable, Encodable};
-use fedimint_core::module::{ExtendsCommonModuleGen, ModuleCommon, TransactionItemAmount};
+use fedimint_core::module::{
+    CommonModuleGen, ExtendsCommonModuleGen, ModuleCommon, TransactionItemAmount,
+};
 use fedimint_core::{apply, async_trait_maybe_send};
 use fedimint_wallet_common::config::WalletClientConfig;
 pub use fedimint_wallet_common::*;
 
+pub trait WalletClientExt {
+    fn get_network(&self) -> Network;
+}
+
+impl WalletClientExt for Client {
+    fn get_network(&self) -> Network {
+        let (_, wallet_client) = wallet_client(self);
+        wallet_client.get_network()
+    }
+}
+
+fn wallet_client(client: &Client) -> (ModuleInstanceId, &WalletClientModule) {
+    let wallet_client_instance = client
+        .get_first_instance(&WalletCommonGen::KIND)
+        .expect("No ln module attached to client");
+
+    let wallet_client = client
+        .get_module_client::<WalletClientModule>(wallet_client_instance)
+        .expect("Instance ID exists, we just fetched it");
+
+    (wallet_client_instance, wallet_client)
+}
+
```

Comment:
Slightly confused about these changes, were they needed?

Reply:
This is just to get the `bitcoin::Network` for the invoice, which is in the `WalletClientConfig`. I think they should be needed once the Wallet client is implemented? If you have a better suggestion for this, I can change it :)

Reply:
This should probably be a) in the global config or b) in the LN config too. We should not make clients dependent on each other. But not a big deal for MVP, maybe add a TODO to refactor.

Reply:
I added  a TODO in the `LightningClientModule` where the `bitcoin::Network` is read.